ssm/ssm/projects/admin.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib import admin
+
+from ssm.projects.models import Project, MembershipModel
+
+
+class MembershipInline(admin.TabularInline):
+    model = MembershipModel
+
+
+class ProjectAdmin(admin.ModelAdmin):
+    list_display = ['name', 'status', 'customer', 'get_members_count']
+    search_fields = ['name']
+    fieldsets = [
+        (None, {'fields': ['name', 'description', 'specification', 'status', 'customer', 'estimation_in_man_hours']}),
+        ('Duration', {'fields': ['start_date', 'end_date']}),
+    ]
+    inlines = [MembershipInline]
+    list_filter = ['customer']
+
+    def get_members_count(self, obj):
+        return len(obj.members.filter(membershipmodel__date_left=None))
```

Comment:
Please always use `count()` instead of `len()` on QuerySets if all you want to do is determine the number of records in the set. It's much more efficient to handle a count at the database level, using SQL's SELECT COUNT(*), and Django provides a count() method for precisely this reason.